src/main/java/bio/terra/service/profile/ProfileService.java

```diff
@@ -185,7 +185,8 @@ public EnumerateBillingProfileModel enumerateProfiles(
    */
   public BillingProfileModel getProfileById(UUID id, AuthenticatedUserRequest user) {
     if (!iamService.hasAnyActions(user, IamResourceType.SPEND_PROFILE, id.toString())) {
-      throw new IamUnauthorizedException("unauthorized");
+      throw new IamUnauthorizedException(
+          "User '" + user.getEmail() + "' does not have access to billing profile '" + id + "'.");
```

Comment:
Do we usually include the user email in IAM error messages? I don't think it can expose any information but the email is always the caller's email so it seems like they'd already have that information.